### PR TITLE
feat(cdp): Add activecampaign hog function

### PR DESCRIPTION
**Pull request by MarconLP**
## Changes

- adds activeCampaign hog function

## How did you test this code?

added tests

### CODE REVIEW

File: posthog/cdp/templates/__init__.py [hunk 1]
```diff
@@ -23,38 +23,40 @@
...
+    salesforce_update,
+    sendgrid,
+    slack,
+    webhook,
+    zapier,
     zendesk,
-    google_cloud_storage,
-    braze,
-    knock,
 ]
```
Comment (MarconLP) [inline posthog/cdp/templates/__init__.py:60]: sorted this alphabetically

Comment (mariusandra) [reply posthog/cdp/templates/__init__.py]: We might want an alternative way to order the list (via some actual usage data?). It's very messy now. I'd also suggest putting the name of the integration as the first word to make it easy to scan.

Comment (MarconLP) [reply posthog/cdp/templates/__init__.py]: I've reverted the change for now.

File: posthog/cdp/templates/activecampaign/test_template_activecampaign.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from posthog.cdp.templates.helpers import BaseHogFunctionTemplateTest
+from posthog.cdp.templates.activecampaign.template_activecampaign import (
+    template as template_activecampaign,
+)
+
+
+def create_inputs(**kwargs):
+    inputs = {
+        "accountName": "posthog",
+        "apiKey": "2f30ab262aa2bc92cc235ca437fc635dac37040d134d2d0c1be1ae0ec6a5b88476ed17b2",
```
Comment (benjackwhite) [inline posthog/cdp/templates/activecampaign/test_template_activecampaign.py:11]: Every time I see a key like this I worry that it is a real one. Can you instead use a placeholder that is obviously not real like just "API_KEY" please

Comment (benjackwhite) [reply posthog/cdp/templates/activecampaign/test_template_activecampaign.py:11]:
```suggestion
        "apiKey": "API_KEY",
```

File: posthog/cdp/templates/activecampaign/test_template_activecampaign.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        )
+
+        assert self.get_mock_fetch_calls()[0] == snapshot(
+            (
+                "https://posthog.api-us1.com/api/3/contact/sync",
+                {
+                    "method": "POST",
+                    "headers": {
+                        "content-type": "application/json",
+                        "Api-Token": "2f30ab262aa2bc92cc235ca437fc635dac37040d134d2d0c1be1ae0ec6a5b88476ed17b2",
```
Comment (benjackwhite) [inline posthog/cdp/templates/activecampaign/test_template_activecampaign.py:39]:
```suggestion
                        "Api-Token": "API_KEY",
```

**Review by benjackwhite — APPROVED** [review]
Approved with a small change request

**Pull request merged**